upsint/cli.py

```diff
@@ -92,12 +92,16 @@ def list_prs(service, repo):
                     "remote tracking branch, date, divergence status and "
                     "whether the branch was merged to master."
                )
-def list_branches():
+@click.option(
+    '--merged-with', type=click.STRING, default="master",
```

Comment:
upsint ignores the option in my repo :(
```
[dhodovsk@shiny packit-tmp]$ upsint list-branches --merged-with master
╒═════════════════╤═══════════════╤═════════════════════╤══╤════════╕
│ master          │ origin/master │ 2019-04-01 11:47:52 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ version-compare │               │ 2019-03-29 15:23:10 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ flask           │               │ 2019-03-26 11:52:27 │  │ merged │
╘═════════════════╧═══════════════╧═════════════════════╧══╧════════╛
[dhodovsk@shiny packit-tmp]$ git checkout -b cheers
Switched to a new branch 'cheers'
[dhodovsk@shiny packit-tmp]$ upsint list-branches --merged-with master
╒═════════════════╤═══════════════╤═════════════════════╤══╤════════╕
│ cheers          │               │ 2019-04-01 11:47:52 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ master          │ origin/master │ 2019-04-01 11:47:52 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ version-compare │               │ 2019-03-29 15:23:10 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ flask           │               │ 2019-03-26 11:52:27 │  │ merged │
╘═════════════════╧═══════════════╧═════════════════════╧══╧════════╛
[dhodovsk@shiny packit-tmp]$ touch this
[dhodovsk@shiny packit-tmp]$ git add this
[dhodovsk@shiny packit-tmp]$ git commit -m "Very important commit"
[cheers b915bc3] Very important commit
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 this
[dhodovsk@shiny packit-tmp]$ upsint list-branches --merged-with master
╒═════════════════╤═══════════════╤═════════════════════╤══╤════════╕
│ cheers          │               │ 2019-04-01 12:23:02 │  │        │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ master          │ origin/master │ 2019-04-01 11:47:52 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ version-compare │               │ 2019-03-29 15:23:10 │  │ merged │
├─────────────────┼───────────────┼─────────────────────┼──┼────────┤
│ flask           │               │ 2019-03-26 11:52:27 │  │ merged │
╘═════════════════╧═══════════════╧═════════════════════╧══╧════════╛
```

But it lists correct branches when removing :)
```
[dhodovsk@shiny packit-tmp]$ upsint remove-merged-branches
Shall we remove these local branches?
* version-compare
* flask
Y/N? 
```

Reply:
The series of commands you posted seem legit to me: what is your expectation?

Reply:
I expected that when adding `--merged-with master` it won't include branch `cheers` in the list.

Reply:
oh, now I get it: it does not work as filter, it works as a query (for the last column) - we can turn it into a filter

Reply:
No need, that was just nitpicking, Now I get it.